2-mandatory/1-create-functions.js

```diff
@@ -3,15 +3,22 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
+function first5(arr) {
+  if (arr.length < 5) {
+    return `Array Too Short`;
+  } else {
+    const newArr = arr.slice(0, 5);
+    return newArr;
+  }
 }
 
 /*
 Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the same elements, except sorted.
 */
-function sortArray() {
+function sortArray(arr) {
+  return arr.sort();
```

Comment:
nice! your solutions look neat and readable, Appolin. 
do you think sort( ) returns a new array?

Reply:
Thank you @DelniaAlipour! I truly appreciate your comments. I think the sort() method returns the original array but sorted. The sort() method also takes callback function in order to sort number properly but in this case, my function only sorts string.